src/manager/components/AddMealForm/AddMealForm.tsx

```diff
@@ -0,0 +1,84 @@
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Box, Button, FormHelperText, Stack, TextField, Typography } from '@mui/material';
+import { useForm } from 'react-hook-form';
+import { AddMealSchema, addMealSchema } from 'src/schemas/addMealSchema';
+
+const useAddMealForm = () => {
```

Comment:
It is easier to use now on bigger screens with the restricted width. You could also consider adding some margin on smaller screens to prevent the edges of inputs touching the edges of the window.